semgrep-core/tests/OTHER/rules/pattern-x-1.yaml

```diff
@@ -0,0 +1,12 @@
+rules:
```

Comment:
I don't know if we already have this, but would be good to have a test for the case that is only `pattern: $X`

Reply:
I will add one!